src/features/decoders/avif/worker/avifDecode.ts

```diff
@@ -10,15 +10,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import avifDecoder, { AVIFModule } from 'codecs/avif/dec/avif_dec';
+import type { AVIFModule } from 'codecs/avif/dec/avif_dec';
 import wasmUrl from 'url:codecs/avif/dec/avif_dec.wasm';
 import { initEmscriptenModule, blobToArrayBuffer } from 'features/worker-utils';
 
 let emscriptenModule: Promise<AVIFModule>;
 
 export default async function decode(blob: Blob): Promise<ImageData> {
   if (!emscriptenModule) {
-    emscriptenModule = initEmscriptenModule(avifDecoder, wasmUrl);
+    const decoder = await import('codecs/avif/dec/avif_dec');
```

Comment:
Oops, we were bundling the decoder even for browsers that supported AVIF

Reply:
Oh good catch!